assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -57,11 +57,13 @@ spec:
     - expr: max without(endpoint, instance, job, pod, service) (kube_node_labels and
         on(node) kube_node_role{role="infra"})
       labels:
+        label_node_role_kubernetes_io: infra
         label_node_role_kubernetes_io_infra: "true"
```

Comment:
Is there a functional difference between these two lines?

Reply:
Other queries in the file use label_node_role_kubernetes_io instead of label_node_role_kubernetes_io_infra.